src/client/pages/Offer/types.ts

```diff
@@ -13,6 +13,14 @@ export interface Address {
   floor?: string
 }
 
+export type Product = {
```

Comment:
Not a big deal, just thinking if it makes to introduce a new "product" entity here - what do you think?

Or is this just another representation of a quote?

Reply:
🤔 I guess I could update [`OfferQuote`](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/client/pages/Offer/types.ts#L24) type by including a `description` and an `image` and use it in `getStandaloneProducts`/`getAdditionalProducts` selectors.